### PR TITLE
A few interface tweaks

### PR DESCRIPTION
**Pull request by botten-ct**
- show full breakdown using a calculated pixelsPerMillisecond
- expand eventDetails on hover
- Add some more margin in the graph as the last '[xx] ms' was cut off
- borders a bit softer on the eyes

### CODE REVIEW

File: src/PerformrRunnerResultGraph.jsx [hunk 1]
```diff
@@ -75,7 +75,12 @@ export default class PerformrRunnerResultGraph extends Component {
     }
 
     render() {
-        const {instanceKey} = this.props;
+        const {instanceKey, resultObject} = this.props;
+        
+        // Calculate pixelsPerMillisecond
+        // (PLOT_EXTRA_PIXELS + PerformrRunnerResultGraph_leftColumnWidth) should be subtracted from the available width
+        const duration = parseInt(resultObject.getIn(['timing', 'duration']), 0);
```
Comment (ghost) [inline src/PerformrRunnerResultGraph.jsx]: `parseInt(foo, 0)` is intended for strings (afaik this value is already a number), and also the last argument will cause it to parse hex and octal strings. Why not use `Math.floor()`

File: src/PerformrRunnerResultGraph.jsx [hunk 2]
```diff
@@ -75,7 +75,12 @@ export default class PerformrRunnerResultGraph extends Component {
...
+        const pixelsPerMillisecond = (screen.availWidth  - 375) / duration;
```
Comment (ghost) [inline src/PerformrRunnerResultGraph.jsx]: `screen.availWidth  - 375` seems like a dirty hack, which is why the linter rejects it ^^ 
Note that this graph is also used fullscreen, namely in Openrunner. Why not introduce some kind of zoom/range prop to `<PerformrRunnerResultGraph/>`?

File: src/components/style/EventPlotItem.scss [hunk 3]
```diff
@@ -30,10 +30,18 @@
 
         &.expanded > .children {
             display: block;
+            -webkit-box-shadow: inset -8px -8px 6px -6px  rgba(0,0,0,0.33), inset -8px 8px 6px -6px  rgba(0,0,0,0.26);
+            -moz-box-shadow: inset -8px -8px 6px -6px  rgba(0,0,0,0.33), inset -8px 8px 6px -6px  rgba(0,0,0,0.26);
```
Comment (ghost) [inline src/components/style/EventPlotItem.scss]: Are those vendor prefixes still needed? I suspect browsers that old will not be able to render this project for many other reasons.

**Review by ghost — CHANGES_REQUESTED** [review]

File: src/PerformrRunnerResultGraph.jsx (hunk 2 above)
Comment (botten-ct) [reply src/PerformrRunnerResultGraph.jsx]: 1. We do need the available width to calculate the pixelsPerMillisecond so the full breakdown is shown (as requested by stakeholder). This is exactly what we needed afaik.

2. Cause we implement changes incremental.

Comment (ghost) [reply src/PerformrRunnerResultGraph.jsx]: Sure, but we have to make sure it also works properly in Openrunner. Which is a more important use case at the moment.
Would everything break, or would we just see white space?

**Comment by ghost** [comment]
Merged https://github.com/computestdev/performr-runner-result-graph/commit/ca6756b50cd7f4bfc0577a9c0f22cdfed00720c7

**Pull request closed without merging**